### PR TITLE
Undefined font corrections

### PR DESCRIPTION
**Pull request by rdswords**
GHI (If applicable): #69 

## Description:
Resolving undefined font16 and font30 by changing them all to font13 (in Estuary skin, font 13 is size 30).

## Checklist:
- [X ] I have based this PR against the develop branch

### CODE REVIEW

File: resources/skins/Main/1080i/script-plex-pre_play.xml [hunk 1]
```diff
@@ -256,6 +257,53 @@
                         <label>$INFO[Window.Property(remainingTime)]</label>
                     </control>
                 </control>
+
+                <control type="grouplist">
```
Comment (pannal) [inline resources/skins/Main/1080i/script-plex-pre_play.xml:261]: What's this?

**Comment by pannal** [comment]
Applying the gist of this for 0.7.0 without merging this due to other changes. Thanks!

File: resources/skins/Main/1080i/script-plex-pre_play.xml (hunk 1 above)
Comment (rdswords) [reply resources/skins/Main/1080i/script-plex-pre_play.xml:261]: It's just shifting the control group code from line 324 to 261 so it's right after the control group for the title/edition stuff (to be easier to keep track of going forward with the ratings moved up to the same line as those items).

Comment (rdswords) [reply resources/skins/Main/1080i/script-plex-pre_play.xml:261]: FYI it does also include the tweaks to alignment/pixel values for the control group.

Comment (pannal) [reply resources/skins/Main/1080i/script-plex-pre_play.xml:261]: Oh OK. I haven't tested those. Please reopen the PR if you want me to re-check :)

Comment (rdswords) [reply resources/skins/Main/1080i/script-plex-pre_play.xml:261]: Would it be easier for now if I modified my fork, to keep that control group at the original line number so just the key changes are highlighted in my commit?

Comment (pannal) [reply resources/skins/Main/1080i/script-plex-pre_play.xml:261]: No idea to be honest.

**Pull request closed without merging**